### PR TITLE
Add basic comment section to portfolio

### PR DESCRIPTION
**Pull request by eash130**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -24,9 +25,23 @@
...
+    jsonValues.add("They said three was fine, but what's wrong with four.");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Eashan!</h1>");
+    String convertedJson = String.format("{\"firstMessage\": \"%s\", \"secondMessage\": \"%s\", " +
+                                         "\"thirdMessage\": \"%s\", \"fourthMessage\": \"%s\"}",
+                                         jsonValues.get(0), jsonValues.get(1), jsonValues.get(2), jsonValues.get(3));
```
Comment (eash130) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: This line violates the 100 column limit as outlined in the style guide. Will Fix.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -24,9 +25,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> jsonValues;
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: We should name this something more descriptive (e.g. comments)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -24,9 +25,23 @@
...
+    jsonValues.add("Let's add another one!");
+    jsonValues.add("Well three's a crowd");
+    jsonValues.add("They said three was fine, but what's wrong with four.");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Eashan!</h1>");
+    String convertedJson = String.format("{\"firstMessage\": \"%s\", \"secondMessage\": \"%s\", " +
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: can you use the gson library here as opposed to writing out the JSON manually?

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -68,3 +68,20 @@ function addRandomQuote() {
 function greetBack() {
   fetch('/data').then(response => response.text()).then(response => document.getElementById('greeting-response').innerHTML = response);
 }
+
+function fetchJson() {
+  fetch('/data').then(response => response.json()).then(messages => {
+    const commentSection = document.getElementById('comment-list');
+    commentSection.appendChild(createListElement(messages.firstMessage));
```
Comment (prabenzom) [inline portfolio/src/main/webapp/script.js]: can you loop through the messages here?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (prabenzom) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Also, i think you should make this a json array of strings

**Review by prabenzom — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -24,9 +26,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> allMessages;
+
+  @Override
+  public void init() {
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:32]: nit: I think just "messages" is fine (or comments)

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -68,3 +68,17 @@ function addRandomQuote() {
 function greetBack() {
   fetch('/data').then(response => response.text()).then(response => document.getElementById('greeting-response').innerHTML = response);
 }
+
+function fetchJson() {
```
Comment (prabenzom) [inline portfolio/src/main/webapp/script.js:72]: nit: I think you can be more descriptive here (fetchComments or fetchMessages).

**Review by prabenzom — APPROVED** [review]

**Pull request merged**